tanzu-community-edition.rb

```diff
@@ -163,7 +179,8 @@ def brew_uninstall_script
   ${HOME}/.config/tanzu/tkg/bom \
   ${HOME}/.config/tanzu/tkg/providers \
   ${HOME}/.config/tanzu/tkg/.tanzu.lock \
-  ${HOME}/.config/tanzu/tkg/compatibility/tkg-compatibility.yaml
+  ${HOME}/.config/tanzu/tkg/compatibility/tkg-compatibility.yaml \
+  ${HOME}/.tanzu
```

Comment:
Is `${HOME}/.tanzu` a directory? I've not noticed it before.

Reply:
It was the original config path before moving to `${HOME}/.config/tanzu`. This was prior to launch though. I would think it wouldn't be needed anymore. I'm guessing it was added here to match our `uninstall.sh` script:

https://github.com/vmware-tanzu/community-edition/blob/main/hack/release/uninstall.sh#L60-L61

Reply:
Yup, you're both right. It's the old TKG install location and has since been removed but kept in because there is logic in the tanzu cli to still recognize this and handle a "migration" to the new folder.

This is probably not needed though since we didn't implement the choco or homebrew until we released to the public.

Reply:
@karuppiah7890 if we can remove that last line and will approve to merge ASAP

Reply:
> It was the original config path before moving to `${HOME}/.config/tanzu`. This was prior to launch though. I would think it wouldn't be needed anymore. I'm guessing it was added here to match our `uninstall.sh` script:
> 
> https://github.com/vmware-tanzu/community-edition/blob/main/hack/release/uninstall.sh#L60-L61

That's right 😅 I just blindly believed that the TCE repo uninstall script must be the right thing and just used it up here without questioning it